qiskit/_qobj.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""Models for QObj and its related components."""
+
+
+class Qobj(object):
+    """ Representation of a Quantum Object.
+
+    Attributes:
+        id_ (str): Qobj identifier.
+        config (QobjConfig): config settings for the Qobj.
+        circuits (list(QobjCircuit)): list of circuits.
+    """
+    def __init__(self, id_, config, circuits=None):
+        self.id_ = id_
```

Comment:
Thanks for catching it! It was actually not for marking it as private, but to avoid shadowing the Python built-in name `id` - I'll at least change the attribute definition, as `self.id` does not clash with the symbol name.